src/lib/index.js

```diff
@@ -350,19 +350,25 @@ lib.noneOrAll = function(containerIn, containerOut, attrList) {
     }
 };
 
-/** merge data array into calcdata items
+lib.mergeArray = function(traceAttr, cd, cdAttr) {
```

Comment:
done in https://github.com/plotly/plotly.js/pull/1761/commits/3c3fda4da845df34f3b730f4f0a2b2b347fd8e6c